zebedee-cms/src/main/java/com/github/onsdigital/zebedee/permissions/service/JWTPermissionsServiceImpl.java

```diff
@@ -0,0 +1,282 @@
+package com.github.onsdigital.zebedee.permissions.service;
+
+import com.github.onsdigital.exceptions.JWTVerificationException;
+import com.github.onsdigital.zebedee.json.CollectionDescription;
+import com.github.onsdigital.zebedee.json.PermissionDefinition;
+import com.github.onsdigital.zebedee.model.Collection;
+import com.github.onsdigital.zebedee.session.model.Session;
+import com.github.onsdigital.zebedee.session.service.Sessions;
+import com.github.onsdigital.zebedee.teams.model.Team;
+import com.github.onsdigital.zebedee.user.model.User;
+import org.apache.commons.lang3.ArrayUtils;
+import org.apache.commons.lang3.StringUtils;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Set;
+
+import static com.github.onsdigital.zebedee.logging.CMSLogEvent.warn;
+
+public class JWTPermissionsServiceImpl implements PermissionsService {
+    static final String PUBLISHER_PERMISSIONS = "publisher";
+    static final String ADMIN_PERMISSIONS = "admin";
```

Comment:
These values will need to match the group names rather than the type of permissions. These are the default groups for admins and publishers which are [`role-admin`](https://github.com/ONSdigital/dp-identity-api/blob/develop/models/groups.go#L14) and [`role-publisher`](https://github.com/ONSdigital/dp-identity-api/blob/develop/models/groups.go#L17) respectively:

```suggestion
    static final String PUBLISHER_GROUP = "role-publisher";
    static final String ADMIN_GROUP = "role-admin";
```